experiments/ClimaEarth/run_amip.jl

```diff
@@ -194,7 +196,7 @@ catch error
         "MODEL.ICE.HAD187001-198110.OI198111-202206_lowres.nc",
     )
 end
-co2_data = joinpath(co2_dataset_path(), "mauna_loa_co2.nc")
+co2_data = joinpath(@clima_artifact("co2_dataset", comms_ctx), "co2_mm_mlo.txt")
```

Comment:
I'm not sure why this change fails. One difference I notice is that now we're passing `comms_ctx` into `@clima_artifact`, where it was `nothing` before. I would expect this to work, so maybe it's a bug in ClimaArtifacts?

Since we don't need to download this artifact lazily, we don't need to provide the context so we could try calling `@clima_artifact` without the context. We should also figure out if this is a bug in ClimaArtifacts or if we're using `@clima_artifact` in a way that isn't meant to be supported

Reply:
It seems like there's a second Artifacts.toml file in experiments/ClimaEarth, and adding @clima_artifacts made it search for the co2 dataset in that file. Because the co2_data is not used anywhere else at the moment, should I remove it from the main Artifact.toml and move it to the Artifacts.toml file in experiments/ClimaEarth? Or should I revert to using co2_dataset_path()? It also seems like the Artifacts.toml file in experiments/ClimaEarth has the landmark artifact, which is also the main  Artifacts.toml, and it is accessed using mask_dataset_path() instead of @clima_artifacts

Reply:
Oh good catch! I don't think we should have two Artifacts.toml files when all the artifacts are used for ClimaEarth experiments. I would recommend adding CO2 to `experiments/ClimaEarth/Artifacts.toml`, then we can get rid of `artifacts/Artifacts.toml` altogether. Our goal is to slim down the `artifacts/` folder as we switch to using ClimaArtifacts, and eventually get rid of the folder

Reply:
CO2 data is now in `experiments/ClimaEarth/Artifacts.toml`. There are only three remaining entries in `artifacts/Artifacts.toml`: land_mask, sic, and sst. I can't find any usage of sic and set. It seems like their usage is replaced by historical_sst_sic. When I move land_mask to ClimaArtifacts I will also get rid of the unused sic and set, which should just remove all files from `artifacts/` except the script that downloads the Atmos and Land artifacts.

Reply:
I think `sic` and `sst` should be removed, they are not needed anymore. I think we also might be at the point where all  land artifact are automatically downloaded (the one missing was the albedo).

Reply:
This sounds good, thank you Teja!